### PR TITLE
Create a Pattern for `Footer 1` Design

### LINKED ISSUE #8952: Create `Footer 1` pattern
Style=Footer 1, Mode=Light

Check Figma designs F5AAbRJOGbTeEweXPFtHgf-fi-1327_26109 for the latest updates and mobile views.

### PR DESCRIPTION
**Pull request by danielwrobert**
WIP: Create a pattern for `Footer 1`.

Fixes #8952 

#### Accessibility

- [ ] I've tested using only a keyboard (no mouse)
- [ ] I've tested using a screen reader
- [ ] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
- [ ] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

#### Other Checks

- [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] This PR adds/removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] I tagged two reviewers because this PR makes queries to the database or I think it might have some security impact.

### Screenshots

| Before | After |
| ------ | ----- |
|        |       |

#### Automated Tests
* [ ] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [ ] E2E tests

#### User Facing Testing

1.
2.
3.

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [ ] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Add suggested changelog entry here.

### CODE REVIEW

File: patterns/footer-1.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->
+<footer class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"var:preset|spacing|40","bottom":"0","left":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-right:var(--wp--preset--spacing--40);padding-bottom:0;padding-left:var(--wp--preset--spacing--40)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"style":{"color":{"duotone":"var:preset|duotone|dark-grayscale"}}} /-->
+
+<!-- wp:navigation {"ref":242} /--></div>
```
Comment (albarin) [inline patterns/footer-1.php]: We need to make sure we don't include any ids or refs in patterns, they will not exist in other stores.
Here we can just use `<!-- wp:navigation /-->`.

File: patterns/footer-1.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * Slug: woocommerce-blocks/footer-1
+ * Categories: WooCommerce
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->
+<footer class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"var:preset|spacing|40","bottom":"0","left":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-right:var(--wp--preset--spacing--40);padding-bottom:0;padding-left:var(--wp--preset--spacing--40)"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"style":{"color":{"duotone":"var:preset|duotone|dark-grayscale"}}} /-->
```
Comment (albarin) [inline patterns/footer-1.php]: I would remove any styling and just use `<!-- wp:site-logo /-->`.

File: patterns/footer-1.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * Title: WooCommerce Footer 1
+ * Slug: woocommerce-blocks/footer-1
+ * Categories: WooCommerce
+ * Block Types: core/template-part/footer
+ */
+?>
+
+<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->
+<footer class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"top":"0","right":"var:preset|spacing|40","bottom":"0","left":"var:preset|spacing|40"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
```
Comment (albarin) [inline patterns/footer-1.php]: I think the footer should probably have full width: `<!-- wp:group {"tagName":"footer","layout":{"type":"default"}} -->` and `<footer class="wp-block-group alignfull">` similar to the other footers we already have, like [footer-simple.php](https://github.com/woocommerce/woocommerce-blocks/blob/6fe5fbb887527242dab9ac7675ea2e54dc365dd8/patterns/footer-simple.php#L10).

**Review by albarin — CHANGES_REQUESTED** [review]
Thanks for working on this 👏 
I left some comments with small tweaks. 
Would be nice also to improve the indentation, it makes it much better to read and spot mistakes (similar to this [footer-simple.php](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/patterns/footer-simple.php)).

For patterns, in general, what we also need to make sure is that they look good on different screen sizes and different theme colors (dark, light, etc.)

File: patterns/footer-1.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Title: WooCommerce Footer 1
```
Comment (albarin) [inline patterns/footer-1.php:3]: Could you check with @shaunandrews how can we improve the title?

File: patterns/footer-1.php (hunk 2 above)
Comment (danielwrobert) [reply patterns/footer-1.php]: Addressed in 7168d70.

File: patterns/footer-1.php (hunk 1 above)
Comment (danielwrobert) [reply patterns/footer-1.php]: Addressed in 4b97232.

File: patterns/footer-1.php (hunk 3 above)
Comment (danielwrobert) [reply patterns/footer-1.php]: Addressed in 16b7481.

**Comment by danielwrobert** [comment]
@albarin I've updated all of the items in your feedback and noted accordingly. I've also updated the indentation for the pattern itself. Thank you for your review!

That being said, I'm going to go ahead and close this PR for now, due to the updates in our designs and plans.

**Pull request closed without merging**